packages/mdc-select/component.ts

```diff
@@ -186,9 +186,8 @@ export class MDCSelect extends MDCComponent<MDCSelectFoundation> {
     // Initially sync floating label
     this.foundation_.handleChange(/* didChange */ false);
 
-    if (this.selectAnchor_.classList.contains(cssClasses.DISABLED)) {
-      this.disabled = true;
-    }
+    // Sets disabled state in foundation
+    this.disabled = this.selectAnchor_.classList.contains(cssClasses.DISABLED);
```

Comment:
Can we move disabled state in foundation? 

`get disabled()` static method can use this disabled state from foundation as source of truth.